rust/extractor/src/main.rs

```diff
@@ -180,11 +182,19 @@ fn cwd() -> anyhow::Result<AbsPathBuf> {
 
 fn main() -> anyhow::Result<()> {
     let start = Instant::now();
+    let flame_layer = if let Ok(path) = std::env::var("CODEQL_EXTRACTOR_RUST_OPTION_FLAME_LOG") {
```

Comment:
I would prefer this to go through the same mechanism we have for other configuration knobs, rather than a direct access to the env here:
* adding the corresponding option to `codeql-extractor.yml` with docs and all
* adding `flame_log` to the `Config` structure
* configuring tracing after extracting said configuration

This would make it work with the whole extractor option machinery, and from the code perspective I like the invariant that all configuration goes through the `Config` structure, and we don't have to go hunt for other env accesses across the code.